Website/src/Foundation/BranchPresets/BranchPresets.config

```diff
@@ -0,0 +1,14 @@
+﻿<!-- Add this to App_Config/Include to enable branch presets -->
```

Comment:
Why do we have 2 copies of this file? With this one having an invalid namespace/assembly name, I think it would be safe to delete it.